packages/fxa-settings/src/pages/ResetPassword/ResetPassword/index.tsx

```diff
@@ -5,7 +5,7 @@
 import { RouteComponentProps, useNavigate } from '@reach/router';
```

Comment:
I thought in #14767 we moved all `pages/ResetPassword/ResetPassword` files into `pages/ResetPassword` but I think we missed these... if your PR diff remains small, you could maybe use `git mv` to help us out there?

Reply:
They've been moved with a recent PR, looks like they're in the right spot on `main`